dune-project

```diff
@@ -0,0 +1,3 @@
+(lang dune 1.1)
+(name syslog-message)
+(version 0.0.2)
```

Comment:
I think this should be moved to a Changelog file which can be then used by dune-release to create the proper tarballs for releases.

Reply:
I don't understand what you mean - `dune-project` is a Dune system file, so move what?

Reply:
Sorry I'm on a train with bad connectivity.

What I meant was to remove the version information from the dune-project file and instead move it to the CHANGELOG.md, which is picked up by dune-release and used for tags, releases and release changelogs by dune-release. This avoids duplicating the version information and forgetting to update it in one place and also simplifies releases and opam-repository PRs.

Reply:
Gotcha - ultimately the plan is that `dune-project` becomes the source of truth for the version and it gets propagated elsewhere, but at the moment `dune-release` does everything that's needed, so I've removed it.